build/tutorials-contest/liquid-avax/.gitignore

```diff
@@ -0,0 +1,3 @@
+./Backend/package.json
```

Comment:
Please remove .gitignore since we won't be having devs install these apps in the same directory as the docs so it's not needed.